nemo/collections/asr/data/audio_to_label.py

```diff
@@ -282,18 +282,32 @@ class _AudioLabelDataset(Dataset):
     def output_types(self) -> Optional[Dict[str, NeuralType]]:
         """Returns definitions of module output ports.
         """
-        return {
+
+        output_types = {
             'audio_signal': NeuralType(
                 ('B', 'T'),
                 AudioSignal(freq=self._sample_rate)
                 if self is not None and hasattr(self, '_sample_rate')
                 else AudioSignal(),
             ),
             'a_sig_length': NeuralType(tuple('B'), LengthsType()),
-            'label': NeuralType(tuple('B'), LabelsType()),
-            'label_length': NeuralType(tuple('B'), LengthsType()),
         }
 
+        if self.is_regression_task:
+            output_types.update(
+                {
+                    'targets': NeuralType(tuple('B'), RegressionValuesType()),
+                    'targets_length': NeuralType(tuple('B'), LengthsType()),
+                }
+            )
+        else:
+
+            output_types.update(
+                {'label': NeuralType(tuple('B'), LabelsType()), 'label_length': NeuralType(tuple('B'), LengthsType()),}
```

Comment:
`label` for regression should be `('B', 'D')` to support multivariate regression in the future (even if D = 1 for now).